cloudigrade/util/tests/helper.py

```diff
@@ -155,6 +155,58 @@ def generate_mock_ec2_instance(instance_id=None, image_id=None, subnet_id=None,
     return mock_instance
 
 
+def generate_mock_image(image_id=None, encrypted=False):
```

Comment:
Does it make sense to use Moto ( https://github.com/spulec/moto ) instead of mocking out AWS objects by hand?